xbout/load.py

```diff
@@ -317,6 +322,19 @@ def attrs_remove_section(obj, section):
         if info:
             warn("No geometry type found, no physical coordinates will be added")
 
+    if grid and ds.options:
+        grididoptions = ds.options.get("grid_id", None)
```

Comment:
I think the `grid_id` should be in the simulation output rather than the options? So it should end up in `metadata`?
```suggestion
    if grid:
        grididoptions = ds.metadata.get("grid_id", None)
```